common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -307,9 +308,9 @@ public MapVisibility getLabelVisibility() {
     }
 
     private static final class PlaceCategory implements MapCategory {
-        private static final MapVisibility PROVINCE_VISIBILITY = new MapVisibility(0, 32, 3);
-        private static final MapVisibility CITY_VISIBILITY = new MapVisibility(0, 74, 3);
-        private static final MapVisibility PLACE_VISIBILITY = new MapVisibility(32, 86, 3);
+        private static final MapVisibility PROVINCE_VISIBILITY = new FullMapVisibility(0, 32, 3);
```

Comment:
We should have a default value for label visibility that is different from the icon visibility, that has a fade of 3 instead of 6.

A note on this value: this was based on the old behavior. I tried setting 6 here as well to get consistency with icon fading, but it actually turned out not to look so good. An icon can fade in slowly over time and still look good, but when text was fading to slowly it just became illegible at low alpha and it just looked plain wrong and bad, so I think we should keep a shorter fade time for labels. I have verified with a synthetic mapfeature which has both a label and an icon (we currently don't have any such, but we could easily get with the new mapdata framework) and it looks good. I was worried that it should look strange with different fade times but it actually turns out very well.

Reply:
> We should have a default value for label visibility that is different from the icon visibility, that has a fade of 3 instead of 6.

Do I need to change anything here? I understand the explanation, but I don't see if anything needs to be done.

Reply:
Yes, these should be like `DerivedVisibilty.withMin(32)`.

Reply:
Done.